assets/parquet/Dockerfile

```
@@ -0,0 +1,68 @@
+WORKDIR /home/presto/presto-native-execution
+RUN mkdir build;
+RUN rm -rf velox;
+RUN git clone https://github.com/anlowee/velox.git;
+WORKDIR /home/presto/presto-native-execution/velox
+RUN git checkout xwei/benchmark-test;
+
+WORKDIR /home/presto
+RUN ./mvnw clean install -DskipTests -pl -presto-docs -T1C;
+
+WORKDIR /home/presto/presto-native-execution/build
+RUN cmake .. && make -j$(nproc) presto_server;
```

Hadolint (2.12.0)

[warning] Quote this to prevent word splitting.

(SC2046)